packages/client/src/checkout/__tests__/deleteCheckoutOrderPromocodes.test.ts

```diff
@@ -0,0 +1,44 @@
+import * as checkoutClient from '../index.js';
+import {
+  type GetCheckoutOrderResponse,
+  OrderStatusError,
+} from '../types/index.js';
+import { id } from 'tests/__fixtures__/checkout/index.mjs';
+import client from '../../helpers/client/index.js';
+import fixtures from '../__fixtures__/deleteCheckoutOrderPromodes.fixtures.js';
+import mswServer from '../../../tests/mswServer.js';
+
+describe('checkout client', () => {
```

Comment:
You can remove this describe section as it is not necessary.

Reply:
This is just to be consistent with what we do in the other files...

Reply:
Okay, I thought other files were not using it.